src/olympia/reviews/tests/test_views.py

```diff
@@ -1482,7 +1501,10 @@ def test_omit_body_and_title_completely_just_rating(self):
         assert review.reply_to is None
         assert review.addon == self.addon
         assert review.version == self.addon.current_version
-        assert response.data['version'] == review.version.version
+        assert response.data['version'] == {
```

Comment:
Hmm, seems odd that so many tests needed updating. Perhaps it's checking the version in too many places? Just thinking out loud, not something to worry about for this patch.